src/main/java/at/hannibal2/skyhanni/features/inventory/HeldTimeInLore.kt

```diff
@@ -0,0 +1,34 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getBottleOfJyrreSeconds
+import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getSecondsHeld
+import at.hannibal2.skyhanni.utils.TimeUtils
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class HeldTimeInLore {
+    private val config get() = SkyHanniMod.feature.inventory
+
+    private val jyrreBottle by lazy { "NEW_BOTTLE_OF_JYRRE".asInternalName() }
+    private val cacaoTruffle by lazy { "DARK_CACAO_TRUFFLE".asInternalName() }
+
+    @SubscribeEvent
+    fun onTooltip(event: LorenzToolTipEvent) {
```

Comment:
No skyblock check?

Reply:
Oh yeah.
but also RestorePieceOfWizardPortal.kt doesn't have it

Reply:
so one bit of bad code means we should add another bit of bad code?

Reply:
Well it's not bad code.
It should work in neu pv, and I don't see a point for another check here?
It can't misfire, since it requires the id to match.